docs/DeveloperGuide.md

```diff
@@ -103,7 +103,7 @@ How the `Logic` component works:
 
 1. When `Logic` is called upon to execute a command, it is passed to an `AddressBookParser` object which in turn creates a parser that matches the command (e.g., `DeleteCommandParser`) and uses it to parse the command.
```

Comment:
Should the AddressBook in `AddressBookParser` here be changed to InternTracker as well? 

I see this in other parts of the code as well!